conda_smithy/templates/README.md.tmpl

```diff
@@ -34,7 +34,7 @@ It is possible to list all of the versions of `{{ package.name() }}` available o
 conda search {{ package.name() }} --channel {{ channel_name }}
 ```
 
-{% if channel_name == 'conda-forge' %}
+{%- if channel_name == 'conda-forge' -%}
```

Comment:
So this isn't quite the right way to do this. It is eliminating all the whitespace between this and the next title. Would pick one side to `-` and then move this line up or down in the document accordingly to avoid further whitespace errors.